unit-tests.gs

```diff
@@ -81,3 +81,29 @@ function testSemVer () {
     assert(v2.compare(v1) === (-comp.result), 'Comparison between ' + comp.v2 + ' and ' + comp.v1 + ' did not return the expected value of ' + (-comp.result));
   });
 }
+
+/**
+ * Test all events from selected persiod. It won't send actual mails to you, but put content of them into log.
+ *
+ * !!! Execution of this method very ofted exceed maximum time.
+ *
+ * @param testDate - First date to start test; by default 01.01.2017.
+ * @param numberOfDaysToTest - Number of days to preform test; by default 1 year.
+ */
+function testSelectedPeriod (testDate, numberOfDaysToTest) {
+
+  testDate = testDate || new Date('2017/01/01 06:00:00');
+  numberOfDaysToTest = numberOfDaysToTest || 365;
+
+  log.add('testWholeYear() running.', 'info');
```

Comment:
This log bears the old name of the function and while we are at it it could be changed to '...running checking from DATE for X days'.